src/commands/moderation/temporaryCommandEdit.ts

```diff
@@ -0,0 +1,339 @@
+import { Colors, EmbedField, TextChannel } from 'discord.js';
+import moment from 'moment';
+import { CommandDefinition } from '../../lib/command';
+import { Roles, Channels, CommandCategory } from '../../constants';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { getConn } from '../../lib/db';
+import TemporaryCommand from '../../lib/schemas/temporaryCommandSchema';
+
+const permittedRoles = [
+    Roles.ADMIN_TEAM,
+    Roles.MODERATION_TEAM,
+];
+
+const helpEmbed = (evokedCommand: String) => makeEmbed({
+    title: 'Manage Temporary Commands - Help',
+    description: 'A command to manage temporary commands that consist of a Title, Content and Color. This can be used to create temporary commands for different purposes.',
+    fields: [
+        {
+            name: 'Adding a Temporary Command',
+            value: makeLines([
+                'To create a temporary command, run the following bot command: ',
+                `\`${evokedCommand} add <command> <severity> <title> <content>\`.`,
+                '`command`: The command to be created that can be executed by regular users.',
+                '`severity`: The type can be `info`, `warning`, or `error`. Depending on the severity, the command output will have an appropriate color: regular, yellow or red.',
+                '`title`: A double quote (`"`) encapsulated string used as the title of the bot message.',
+                '`content`: A double quote (`"`) encapsulated string used as the content of the bot message.',
+                'Example:',
+                `\`${evokedCommand} add good-news-everyone info "Good News Everyone!" "The Experimental version is the greatest in the world!"\``,
+                '\u200B',
+            ]),
+            inline: false,
+        },
+        {
+            name: 'Show a Temporary Command',
+            value: makeLines([
+                'To show an existing temporary command, run the following bot command: ',
```

Comment:
```suggestion
                'To show an existing temporary command with logging details, use the following bot command: ',
```

Want to make it clear it has 'extra details' not sure I have the best wording